src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -141,7 +141,7 @@ public function formatAndWrap(string $message, int $width)
     {
         $offset = 0;
         $output = '';
-        $tagRegex = '[a-z][a-z0-9,_=;-]*+';
+        $tagRegex = '[a-z][^<>]*+';
```

Comment:
Why is it needed?

Reply:
To accept URLs, check test case at https://github.com/symfony/symfony/pull/29168/files#diff-fabb4589bcf7453c1158e6a61a9c39f3R258